### DISCUSSION TITLE
[NEW UI] Replace all Sunnyside designs.

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2301] [Ideas]
I know that Daniel's design was very important for the development and growth of this game, tut I believe now it can hurt the growth. There are a bunch of games that share the same design, which can bring some confusion to give the game an identity. This design costs $6 and it was free for a limited time.

I recieve at least 2 ads / day with other games using same design.

I know the sunflower team is working on marketing campaigns. Don't you think that a bad experience with the design in other games could confuse potential new players?

We've already done campaigns for new designs why not replace the old ones?

**Comment by TheDcol** [comment]
Backlog for artists will be insane lol

**Comment by adamhannigan** [comment]
We are slowly shifting away from this pack - you may of noticed all of the recent changes in Open Sea lately.

The base assets form 90% of our background maps which is a significant amount of artwork. We will get there, but it will require a bit of work.

**Reply by future2back** [reply]
Hello @adamhannigan! Thanks for replying. I think the SunflowerLand  team should put an action plan in place right now. I believe we already have the financial resources for this and a community that has already shown itself capable of doing great things. We just need a plan and a deadline, which in my point of view, should be before any marketing action.
We need a unique design, an attractive identity to catch the attention of potential new players. First impression remains, and that's what we should think on.

If we discuss and define an identity with community, this project will follow in parallel with other projects.

We have to work with this community of great artists that we have to our advantage.
Could you please start organizing it?

Thank you.
